auth/register.html

```diff
@@ -1,18 +1,24 @@
 {{define "content"}}
-	<h2>Register</h2>
-	<form method="post">
-		<div class="form-group">
-			<label for="email">Email</label>
-			<input class="form-control" type="text" id="email" name="email" />
+	<div class="container">
+		<h1 class="welcome">Reach your community</h1>
```

Comment:
This is the same as in the `Login` and can easily be a part of a header template snippet.